benchmarks/src/jmh/java/jetbrains/exodus/benchmark/persistit/JMHPersistItTokyoCabinetBenchmarkBase.java

```diff
@@ -81,7 +81,7 @@ private void createEnvironment() throws IOException, PersistitException {
         props.setProperty("datapath", temporaryFolder.getRoot().getAbsolutePath());
         props.setProperty("logpath", "${datapath}/log");
         props.setProperty("logfile", "${logpath}/persistit_${timestamp}.log");
-        props.setProperty("buffer.count.8192", "10");
+        props.setProperty("buffer.count.8192", "5000");
```

Comment:
BTW, performance extremely depends on this setting. E.g., results of random read/write differ by an order of magnitude.